waltz-ng/client/data-types/routes.js

```diff
@@ -0,0 +1,42 @@
+const baseState = {
+    resolve: {
+        dataTypes: [
+            'DataTypesService',
+            (dataTypesService) => dataTypesService.loadDataTypes()
+        ]
+    }
+};
+
+
+const listState = {
+    url: 'data-types',
+    views: {'content@': require('./data-type-list') }
+};
+
+
+const viewByCodeState = {
+    url: 'data-types/code/{code}',
+    views: {'content@': require('./data-type-view') },
+    redirectTo: 'main.data-type.view',
```

Comment:
I'll have another go, it wasn't working when i tried it. perhaps i was
missing something.


On Thu, Aug 11, 2016 at 5:46 PM, David Watkins <notifications@github.com>
wrote:

> In waltz-ng/client/data-types/routes.js
> <https://github.com/khartec/waltz/pull/417#discussion_r74458701>:
>
> > +            (dataTypesService) => dataTypesService.loadDataTypes()
> > +        ]
> > +    }
> > +};
> > +
> > +
> > +const listState = {
> > +    url: 'data-types',
> > +    views: {'content@': require('./data-type-list') }
> > +};
> > +
> > +
> > +const viewByCodeState = {
> > +    url: 'data-types/code/{code}',
> > +    views: {'content@': require('./data-type-view') },
> > +    redirectTo: 'main.data-type.view',
>
> y, but we could do that little dance in the resolver couldn't we ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/khartec/waltz/pull/417/files/e4bc887bdadc42f03d5e44e0876776ce30ea1a5c#r74458701>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC4520fEe4I_EEmJt7hJLovyeEwqIlN5ks5qe1HTgaJpZM4JiBOI>
> .
>



-- 


*Kamran Saleem*
t  +44 (0) 7732 078654
e  kamran@thinkincode.co.uk
w thinkincode.co.uk